hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenAirdropHandlerTest.java

```diff
@@ -563,6 +624,45 @@ void updateUpdatesExistingAirdrop() {
         Assertions.assertThat(tokenValue).isEqualTo(airdropValue.amount() + newAirdropValue.amount());
     }
 
+    @Mock(strictness = LENIENT)
+    protected PreHandleContext preHandleContext;
+
+    @Test
+    void checkPreHandler() {
```

Comment:
Nit: hard to see what this test is doing at a glance. Maybe `prehandleVerifiesValidAirdrop` or similar

Reply:
good point. fixing.